types/index.ts

```diff
@@ -25,6 +25,17 @@ export type SpeakersT = Base & Link;
 
 export type DonationV1T = Base & Link;
 
+export type MerchGeneralT = Base & Link;
+
+export type PartnersT = {
+  title: string;
+  companies: {
```

Comment:
We need to add 1 extra field, **priority** or something like that to determine most valuable partners (for example EPAM should be with priority 1 and should go first in the list)